src/build.bat

```diff
@@ -9,6 +9,8 @@ pyinstaller --noconfirm --log-level=FATAL --noconsole --onefile --hidden-import=
 ECHO Copying files to dist...
 :: Copy settings.ini to dist
 COPY settings.ini dist\settings.ini
+:: Copy logging.yaml to dist
```

Comment:
Comment refers to a yaml, but the file is a JSON file.

Reply:
Good catch! Resolved in da65d5f.